File: src/PolarConverter.JSWeb/App/Models/uploadViewModel.ts
module PolarConverter {
    "use strict";

    export interface UploadViewModel {
        uid?: string;
        gender?: string;
        weight?: number;
        weightMode?: string;
        timeZoneOffset?: number;
        forceGarmin: boolean;
        polarFiles: PolarConverter.PolarFile[];
		age: number;
    }
}

File: src/PolarConverter.BLL/Entiteter/UploadViewModel.cs
namespace PolarConverter.BLL.Entiteter
{
    public class UploadViewModel
    {
        public string Uid { get; set; }
        public double Weight { get; set; }
        public string WeightMode { get; set; }
        public double TimeZoneOffset { get; set; }
        public double V02Max { get; set; }
        public int Age { get; set; }
        public string Gender { get; set; }
        public bool ForceGarmin { get; set; }
        public PolarFile[] PolarFiles { get; set; }
        public string Notes { get; set; }
    }
}